### DISCUSSION TITLE
Ads in Shorts

### DISCUSSION THREAD

**Discussion by POIZ87** [ReVanced/revanced-discussions#1058] [Question] [answered]
I don't now why. I turnt of everything I could. But I still get ads every 3 to 7 shorts. How do I turn this of.

And may I support your German translation? It's not good. And often it makes no sense and I have to guess what it could mean.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
You are welcome and invited to improve the German language translations.  Visit http://translate.revanced.app and create a Crowdin account to contribute.

You can turn off Shorts ads by enabling `Switch create with notifications` as explained [here](https://github.com/ReVanced/revanced-patches/issues/219#issuecomment-2000077995).

**Reply by POIZ87** [reply]
Thanks! I appreciate the quick and helpful answer! 

I will do. Is there a documentation what each single setting do. To find the most appropriate translation. Like a bevor and after?

**Reply by LisoUseInAIKyrios** [reply]
There is talk of adding larger documentation and/or screenshots for patches, but nothing has been added yet.

For now you can check [here](https://translate.revanced.app/project/revanced/discussions/12) for screenshots of what the various settings do.  See the other discussion posts for suggestions and tips on contributing.

**Reply by POIZ87** [reply]
I will do thank you very much.
